torchprime/launcher/cli.py

```diff
@@ -273,7 +284,7 @@ def run(args, name: str | None, use_hf: bool, use_local_wheel: bool):
       "--tpu-type",
       config.tpu_type,
       "--num-slices",
-      str(config.num_slices),
+      str(num_slices),
```

Comment:
how many slices do we test on? would be nice to do both 1 and 2 slices by deafult.

Reply:
Yes, we used to test on 1 slice and now this PR will add both 1 slice and 2 slice testing.